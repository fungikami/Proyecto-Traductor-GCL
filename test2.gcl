|[
  declare
    x, y : int;
    y : int

  // Declarar dos o más veces la misma variable (redeclaración de variables).
  x := 1;
  y := 3;
  p := x + y
]|