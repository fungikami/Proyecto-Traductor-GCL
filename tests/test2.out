c_{34} (c_{34} (c_{24} (c_{20} (c_{31} c_{40} c_{40}))(c_{19} (\lambda x_{3}.c_{13} (\lambda x_{0}.c_{13} (\lambda x_{1}.c_{15} (c_{31} (c_{33}(c_{21} ((c_{5}(c_{9}) (c_{8}))) x_{0}))(c_{33}(c_{21} (x_{1}) x_{0}))) x_{3}))) (\lambda x_{3}.c_{32} (c_{32} c_{37} c_{36}) (c_{32} c_{37} c_{36})))) (c_{67})) (c_{24} (c_{20} (c_{31} c_{40} c_{40}))(c_{19} (\lambda x_{2}.c_{13} (\lambda x_{0}.c_{15} (c_{31} (c_{33}(c_{54} c_{43} (c_{54} c_{44} c_{45})))(c_{33}(x_{0}))) x_{2})) (\lambda x_{2}.c_{32} (c_{36}) (c_{36}))))